demos/common/performance_metrics/metrics.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <chrono>
+
+#include "../samples/ocv_common.hpp"
```

Comment:
I'm not sure about `"`. I think folders in common are supposed to be different libraries. They are like third party libraries to each other.

So you should use `<>`. And should't use relative include paths. Modify `target_include_directories` for `performance_metrics` in cmake instead.

Reply:
performance_metrics are part of common now. You should use quotes.